source/_patterns/02-organisms/50-borrower-dashboard/05-auto-repayments.mustache

```diff
@@ -6,7 +6,17 @@
 
 	<p>
 	{{#hasBillingAgreement}}
-		You have an active billing agreement!
+
+		{{^hasAutoRepayment}}
+			<div class="button activate-auto-repayments-button">
+				Set up automatic repayments
+			</div>
+		{{/hasAutoRepayment}}
+
+		{{#hasAutoRepayment}}
+			you have an active auto-repayment.
```

Comment:
@zvibo This is just stubbing out for now right? If so, then ignore this, but was just thinking "you" should be capitalized "You". Otherwise +1 and going to go ahead and merge